configs/kapt/v2/kapt-reseach-v2.md

```diff
@@ -0,0 +1,42 @@
+
+## KAPT Reseacrh with Java/Koltin mix
+
+
+* Date - 2018 - 5 - 21
+* Tools Version
+
+`
+ "buildSystemConfig": {
+ "buildSystemVersion": "4.7",
+  "agpVersion": "3.1.2",
+      "kotlinVersion": "1.2.41"
+    },
+`
+
+#### Profiling
+The profiling was done with gradle-profiler. Each configuration, except `KotlinOnly`, 
+was profiled 4 times. The `KotlinOnly` configuration was profiled twice, because 
+the gradle-profiler does not support incremental build profiling for Kotlin.
+
+Clean build with build scan profiler
+Incremental build with build scan profiler
+Clean build with `--benchmark` option (second for “KotlinOnly" configuration)
+Incremental build with `--benchmark` option. 
+* [Profiler configs](https://github.com/android/android-studio-poet/blob/master/configs/kapt/v2/gradle-profiler.scenarios) 
```

Comment:
Can you rename these two points a bit? "Profiler scenarios" and "AS Poet configs"

Reply:
Just to stay consistent with terminology.

Reply:
Fixed